Header header
string time
int32 track_number
